src/app_service/service/transaction/router_transactions_dto.nim

```diff
@@ -153,7 +153,7 @@ proc toSendDetailsDto*(jsonObj: JsonNode): SendDetailsDto =
   if jsonObj.getProp("packId", tmpObj):
     let packId = stint.fromHex(UInt256, tmpObj.getStr)
     result.packId = $packId
-  if jsonObj.getProp("communityParams", tmpObj):
+  if jsonObj.getProp("communityParams", tmpObj) and tmpObj.kind == JObject:
```

Comment:
What is this extra `tmpObj.kind == JObject` check for? :thinking:

Reply:
@caybro it checks if the json object is nil or not and if it's nil we don't proceed with the conversion, but only if it's of an object type.